dlme_airflow/utils/dataframe.py

```diff
@@ -32,6 +32,11 @@ def dataframe_to_file(collection):
         .get("name_in_dataframe", "id")
     )
 
+    # We need to set last_harvest_start_date here since the catalog metadata comes
+    # from the YAML Intake catalog file, but in some cases (oai driver currently)
+    # we need the Intake driver to know about the last time the harvest ran successfully.
+    setattr(collection.catalog, "last_harvest_start_date", last_harvest_start_date)
+
```

Comment:
Felt like a hack, but this is the only way I could figure out how to do it, since the Catalog instance has already been populated using metadata from the Intake catalog.